packages/yourloops/test/integration/utils/mockTeamAPI.ts

```diff
@@ -0,0 +1,9 @@
+import TeamAPI from '../../../lib/team/team-api'
+import { TeamMemberRole, TeamType } from '../../../models/team'
+import { UserInvitationStatus } from '../../../models/generic'
+import { HcpProfession } from '../../../models/hcp-profession'
+
+export const mockTeamAPI = () => {
```

Comment:
I don't think that this code will be easy to reuse for more team specific tests, but for now it will do :+1: